TVProvider/TVSpielfilm.py

```diff
@@ -34,9 +34,7 @@ def getProgram(self, session: DialogSession, channels: list) -> list:
 				if '@url' in item.get('enclosure', ''):
 					entry['Image'] = f"<img src={item['enclosure']['@url']} />"
 
-				entry['Time'], entry['Channel'], entry['Show'] = map(
-					str.split, item['title'].split(" | ", maxsplit=2)
-				)
+				entry['Time'], entry['Channel'], entry['Show'] = item['title'].split(" | ", maxsplit=2)
```

Comment:
are you sure this is correct? I thinkt thats something I screwed up and should have been:
```python
entry['Time'], entry['Channel'], entry['Show'] = map(str.strip, item['title'].split(" | ", maxsplit=2))
```
instead or is the stripping not required?

Reply:
The "input" is: <title>18:45 | RTL | RTL Aktuell</title>
It is split by " | " - with spaces. Therefore no additional spaces have to be removed by the strip (so far).
Tests worked fine.

Reply:
ah ok just wondered because I think it was using strip before I refactored it :)